### ISSUE TITLE
Missing list formatting in the text editor - same for line spacing

### ISSUE THREAD

**Issue by handkerchief333** [nextcloud/mail#9330]
### Steps to reproduce

1. Create an e-mail message
2. Create lists in the text editor
3. Create line spaces, single and larger (e.g. Enter or Shift + Enter)

### Expected behavior

The same list in the e-mail editor actually looks like this:
![create-message-list-true](https://github.com/nextcloud/mail/assets/41755733/b1d65e17-c0ee-43a1-bafe-bae10935770b)

This is how the distances are in reality, e.g. in Word etc.:
![create-message-space-true](https://github.com/nextcloud/mail/assets/41755733/5f27e49c-f579-4efe-a5e5-3f0ebb2f05b0)

### Actual behavior

This is how it looks in the editor, which is very confusing and leads to formatting errors when sending, as you cannot see the indentation when composing. 
![create-message-list](https://github.com/nextcloud/mail/assets/41755733/a2fe9f4c-09e9-473b-b895-f198dee01a32)

This is what it looks like when writing:
![create-message-space](https://github.com/nextcloud/mail/assets/41755733/d3a89dc4-79d0-4e36-a288-c7fb22fcf62f)

### Mail app version

3.5.4

### Operating system

FreeBSD 13.2

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database

MariaDB

### Additional info

It would be great if the text editor generally displayed the same formatting as the e-mail is displayed in view mode.

I'm also thinking of single and multiple line breaks, which can also cause problems. When composing an e-mail, you cannot see visually whether you have made only one or a larger line spacing (e.g. Enter or Shift + Enter).

**Comment by ChristophWurst** [comment]
> It would be great if the text editor generally displayed the same formatting as the e-mail is displayed in view mode.

I agree but this is technically challenging. When composing, the CSS of Nextcloud applies and gives the text a certain style. Then, once received, it's up to the engine of the email program to render the HTML again.
To get a more deterministic style while composing and reading the sent email we probably have to add our own css in the editor and embed that same css in the HTML email.

**Comment by ChristophWurst** [comment]
Related: https://github.com/nextcloud/server/issues/43393

**Comment by handkerchief333** [comment]
I understand that technically. It doesn't have to be 100% exactly the same either, as close as possible is enough. Currently, the edit view and the final view are too far apart.

**Comment by handkerchief333** [comment]
The latest updates have made it even worse. Now you no longer see any bullet points or indentations when composing e-mails.

This is what the list looks like in the e-mail editor
![list-create](https://github.com/nextcloud/mail/assets/41755733/96635faa-260b-4eea-a9ff-49f6e5dc1cff)

And this is how the list is formatted and looks to the recipient
![list-should-be](https://github.com/nextcloud/mail/assets/41755733/2f43aeab-c9c0-4b1b-8b89-3a435d3dfb1e)

Mail version: 3.6.0
Nextcloud: 28.0.4

**Comment by alexanderdd** [comment]
maybe related? https://github.com/nextcloud/mail/issues/9858
